src/components/PageBanner/_PageBanner.scss

```diff
@@ -56,3 +56,16 @@
     margin-top: $coa-space-level6;
   }
 }
+
+.coa-PageBanner__container {
+  img {
+    position: absolute;
+    z-index: -1;
+    top: auto;
+    height: 30rem;
```

Comment:
any way to use one of the height variables here? https://github.com/cityofaustin/janis/blob/7d109b4e6b6c70aab784585f37d1a9539edad7cf/src/css/base/_variables.scss#L138
Or if we expect it to vary from the hero on the home page, we could add a height variable in order to be consistent?